### PR TITLE
UCP/ADDRESS: Pass system device id during wireup

### PR DESCRIPTION
**Pull request by yosefe**
# Why
To calculate the expected bandwidth of rendezvous protocols considering both local **and remote** system topology

# How
- Pass system device id in UCP address
- Save remote system device id in endpoint configuration key

### CODE REVIEW

**Comment by yosefe** [comment]
@brminich @evgeny-leksikov can you pls take a look?

File: src/ucp/wireup/wireup_cm.c [hunk 1]
```diff
@@ -330,6 +331,12 @@ static ucs_status_t ucp_cm_ep_init_lanes(ucp_ep_h ep,
     return status;
 }
 
+static unsigned ucp_cm_pack_flags(ucp_worker_h worker)
```
Comment (brminich) [inline src/ucp/wireup/wireup_cm.c:334]: maybe `ucp_worker_cm_pack_flags`?

Comment (yosefe) [reply src/ucp/wireup/wireup_cm.c:334]: most of the funcs in this file are called "ucp_cm_.."

Comment (evgeny-leksikov) [reply src/ucp/wireup/wireup_cm.c:334]: `ucp_cm_worker_address_pack_flags` or shorter `ucp_cm_address_pack_flags`?

Comment (yosefe) [reply src/ucp/wireup/wireup_cm.c:334]: `ucp_cm_address_pack_flags` - sounds good

**Review by evgeny-leksikov — APPROVED** [review]

**Comment by yosefe** [comment]
bot:pipe:retes

**Comment by evgeny-leksikov** [comment]
bot:pipe:retest

**Pull request merged**